app/Spago.hs

```diff
@@ -189,6 +189,13 @@ getAllDependencies Config { dependencies = deps, packages = pkgs } =
               let newAcc = List.foldl' go acc innerDeps
               Map.insert dep x newAcc
 
+-- | Get all the package names from the configuration
+getPackageNames :: Config -> [Text]
```

Comment:
Could we move this to a `where` block in the `available` function? (since it's the only place using it)

Reply:
Done :)